test/clt-tests/integrations/fluentbit.rec

```diff
@@ -0,0 +1,59 @@
+––– input –––
+set -b
+––– output –––
+––– input –––
+export PATH=/usr/bin:/usr/local/bin:/usr/sbin:/sbin:/bin
+––– output –––
+––– input –––
+apt-get update > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+apt-get -y install gnupg2 openjdk-8-jre-headless curl jq > /dev/null; echo $?
+––– output –––
+debconf: delaying package configuration, since apt-utils is not installed
+0
+––– input –––
+mkdir /usr/share/fluentbit && cd /usr/share/fluentbit
+––– output –––
+––– input –––
+wget -P fluentbit http://mirrors.kernel.org/ubuntu/pool/main/liby/libyaml/libyaml-0-2_0.2.2-1_amd64.deb 2>&1 | grep "'fluentbit/libyaml-0-2_0.2.2-1_amd64.deb’ saved"
```

Comment:
The grep filter looks redundant since it doesn't match anything.

Reply:
Fixed